splunk_add_on_ucc_framework/generators/xml_files/create_inputs_xml.py

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright 2024 Splunk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from splunk_add_on_ucc_framework.generators.xml_files import XMLGenerator
+from splunk_add_on_ucc_framework.global_config import GlobalConfig
+from typing import Any, Dict, Union
+from splunk_add_on_ucc_framework import data_ui_generator
+
+
+class InputsXml(XMLGenerator):
+    __description__ = (
+        " Generates inputs.xml based on inputs configuration present in globalConfig,"
```

Comment:
NIT: Space at the beginning of the string

Reply:
Removed the space.